linux_os/guide/system/accounts/accounts-restrictions/root_logins/accounts_root_gid_zero/tests/bad_value.fail.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# remediation = none
+
+sed -i "s#^root*#root:x:0:10:/root:/bin/bash#" /etc/passwd
```

Comment:
When I execute it on a RHEL8 VM the test suite is stuck there and asks for a password.

```
[jcerny@thinkpad scap-security-guide{pr/7685}]$ python3 tests/test_suite.py rule --no-reports --libvirt qemu:///system ssgts_rhel8 accounts_root_gid_zero
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2021-10-04-1101/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_accounts_root_gid_zero
root@192.168.122.70's password: 
```

Also in the GitHub Actions run it has some problems with execution:

```
Run tests/test_rule_in_container.sh --dontclean --logdir logs_bash --remediate-using bash --name ssg_test_suite --datastream ssg-rhel8-ds.xml accounts_root_gid_zero
INFO - The base image option has been specified, choosing Podman-based test environment.
INFO - Logging into logs_bash/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_accounts_root_gid_zero
ERROR - Script bad_value.fail.sh using profile (all) found issue:
ERROR - Rule xccdf_org.ssgproject.content_rule_accounts_root_gid_zero has not been evaluated! Wrong profile selected in test scenario?
ERROR - The initial scan failed for rule 'xccdf_org.ssgproject.content_rule_accounts_root_gid_zero'.
ERROR - Script correct_value.pass.sh using profile (all) found issue:
ERROR - Rule evaluation resulted in fail, instead of expected pass during initial stage 
ERROR - The initial scan failed for rule 'xccdf_org.ssgproject.content_rule_accounts_root_gid_zero'.
Setting console output to log level INFO
```

Reply:
So I believe that changing roots GID is causing this issue, not sure how we want to solve it.

Reply:
Maybe we will have to remove the test scenario and leave this untested.

Reply:
I have removed the test.